middleware/package.json

```diff
@@ -11,7 +11,7 @@
     "build": "node_modules/typescript/bin/tsc -p .",
     "lint": "eslint './*.ts' 'test/**/*.ts' --ext .ts",
     "start": "node dist/index",
-    "test": "nyc --extension .ts mocha --forbid-only -r ts-node/register 'test/unit/**/*.ts'",
+    "test": "cross-env USER_FUNCTION_URI=../middleware/test/fixtures/functions/mock nyc --extension .ts mocha --forbid-only -r ts-node/register 'test/unit/**/*.ts'",
```

Comment:
In case anyone is wondering.  `cross-env` makes setting ENV vars work on both Windows/Linux.

Reply:
😎